qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -476,34 +499,70 @@ def export_to_file(
         file_type=FileType.CSV.value,
         coordinates=Coordinate.CYLINDRICAL.value,
     ):
-        """Prepares and saves the driven control in a file.
+        """
+        Prepares and saves the driven control in a file.
 
         Parameters
         ----------
-        filename : str, optional
+        filename : str
             Name and path of the file to save the control into.
-            Defaults to None
         file_format : str
-            Specified file format for saving the control. Defaults to
-            'Q-CTRL expanded'; Currently it does not support any other format.
-            For detail of the `Q-CTRL Expanded Format` consult
-            `Q-CTRL Control Data Format
-            <https://docs.q-ctrl.com/wiki/output-data-formats#q-ctrl-hardware>` _.
+            Specified file format for saving the control. Defaults to 'Q-CTRL expanded'. Currently
+            does not support any other format. For detail of the Q-CTRL expanded format, see below.
         file_type : str, optional
-            One of 'CSV' or 'JSON'; defaults to 'CSV'.
+            One of 'CSV' or 'JSON'. Defaults to 'CSV'.
         coordinates : str, optional
-            Indicates the co-ordinate system requested. Must be one of
-            'cylindrical', 'cartesian'; defaults to 'cylindrical'
-
-        References
-        ----------
-        `Q-CTRL Control Data Format
-        <https://docs.q-ctrl.com/wiki/output-data-formats#q-ctrl-hardware>` _.
+            The coordinate system in which to save the control. Must be 'cylindrical' or
```

Comment:
Looking at https://qeng-1118.docs.q-ctrl.com/references/python/qctrl-open-controls/qctrlopencontrols.html
This method appears in the middle of other properties. I guess the doc is using the alphabetical order.
Would it be possible to group things into `properties` and `methods`?

Reply:
Hm, I agree it looks a bit strange, but I can't see any option to change it from being alphabetical (https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html).

Reply:
Thanks for looking at it. I guess this is one of the things (e.g., the structure of the doc) we can fix later for the doc as this task is to move the necessary part from Wiki to the doc.